capstone-frontend/app/screens/SignInScreen.js

```diff
@@ -14,6 +14,8 @@ export default function SignInScreen() {
 
   })
 
+  const [formErrors,setFormErrors]= useState({});
```

Comment:
There is already a errors variable present. Please use that or remove that and then use this.